DevelopGitHub/GitHubExtension/GitHubExtension.Tools.LocalizationTool/Model/TranslationDataRow.cs

```diff
@@ -0,0 +1,94 @@
+﻿using GitHubExtension.LocalizationTool.Translate;
+using GitHubExtension.LocalizationTool.ViewModel;
+
+namespace GitHubExtension.LocalizationTool.Model
+{
+    public class TranslationDataRow : NotifyProperyChangedBase
+    {
+        private string _name;
+
+        private string _ru;
+
+        private string _uk;
+
+        private string _en;
+
+        public TranslationDataRow()
+        {
+        }
+
+        public TranslationDataRow(string name)
+        {
+            Name = name;
+        }
+
+        public string Name
+        {
+            get { return _name; }
+            set { CheckPropertyChanged(ref _name, value); }
```

Comment:
https://github.com/ITsvetkoFF/Kv-013/pull/67#discussion-diff-61565110
I thought we decided refuse to use this method